### PR TITLE
#3943 Redesign header and footer

### PR DESCRIPTION
**Pull request by drakkein**
This PR redesigns header and footer components.

To be completed this PR requires assistance from designers to remove empty padding from SVG's. 

### Desktop

**Authorized user**
192 168 8 189_9090_dashboard

**Platform main page**
192 168 8 189_9090_dashboard (1)

**Log in page**
192 168 8 189_9090_dashboard (2)

**Sign up page**
192 168 8 189_9090_dashboard (3)

---

### Mobile

**Authorized user**
image

**Platform main page**
image

**Log in page**
image

**Sign up page**
image

### CODE REVIEW

File: packages/design-system/src/components/buttons/Button.tsx [hunk 1]
```diff
@@ -24,6 +24,7 @@ enum EButtonSize {
   NORMAL,
   SMALL = styles.buttonSmall,
   HUGE = styles.buttonHuge,
+  DYNAMIC = styles.buttonDynamic,
```
Comment (drakkein) [inline packages/design-system/src/components/buttons/Button.tsx]: This ButtonSize performs like `SMALL` for mobile devices and as `NORMAL` for higher screen resolutions

File: packages/design-system/src/styles/mixins/typography.scss [hunk 2]
```diff
@@ -193,6 +186,13 @@
   line-height: 1.5;
 }
 
+@mixin nds-body-small {
```
Comment (drakkein) [inline packages/design-system/src/styles/mixins/typography.scss]: This was renamed to a name from styleguide

**Comment by andywalton** [comment]
@drakkein -> Let's chat about the SVG empty padding tomorrow morning. Generally for the UI icons we're positioning an icon within a 24x24px square. It allows us to easily swap out icons in our design components.

**Comment by andywalton** [comment]
@drakkein -> this is very good. I have a minor request that I omitted from this ticket, apologies. The hover colours for the Footer links go dark, this worked previously but now with the dark footer background they become fairly invisible.

Could you change the link hover colour to: `#C6D30D;` as per the screengrab below:

I'll update the styleguide with this example.

Screenshot 2020-02-10 16 23 45

**Pull request merged**